src/Vision/V1/ImageAnnotatorClient.php

```diff
@@ -30,238 +30,10 @@
 
 namespace Google\Cloud\Vision\V1;
 
-use Google\GAX\AgentHeaderDescriptor;
-use Google\GAX\ApiCallable;
-use Google\GAX\CallSettings;
-use Google\GAX\GrpcConstants;
-use Google\GAX\GrpcCredentialsHelper;
-
 /**
- * Service Description: Service that performs Google Cloud Vision API detection tasks over client
- * images, such as face, landmark, logo, label, and text detection. The
- * ImageAnnotator service returns detected entities from the images.
- *
- * EXPERIMENTAL: this client library class has not yet been declared beta. This class may change
- * more frequently than those which have been declared beta or 1.0, including changes which break
- * backwards compatibility.
- *
- * This class provides the ability to make remote calls to the backing service through method
- * calls that map to API methods. Sample code to get started:
- *
- * ```
- * try {
- *     $imageAnnotatorClient = new ImageAnnotatorClient();
- *     $requests = [];
- *     $response = $imageAnnotatorClient->batchAnnotateImages($requests);
- * } finally {
- *     $imageAnnotatorClient->close();
- * }
- * ```
- *
- * @experimental
+ * {@inheritdoc}
  */
-class ImageAnnotatorClient
+class ImageAnnotatorClient extends ImageAnnotatorGapic
```

Comment:
It might be problematic to have an arrangement where an API being a partial gapic means that the auto-generated client is in a different class than if the API is full GAPIC; transitioning from one state to the other could be disruptive.